src/modules/uORB/uORBDeviceNode.hpp

```diff
@@ -195,6 +196,12 @@ class uORB::DeviceNode : public cdev::CDev, public ListNode<uORB::DeviceNode *>
 	int get_priority() const { return _priority; }
 	void set_priority(uint8_t priority) { _priority = priority; }
 
+	// add item to list of work items to schedule on node update
+	bool register_work_item(px4::WorkItem *item);
```

Comment:
Why are work items added within uORB? What's the connection here?

Reply:
I'll explain separately.